### ISSUE TITLE
Base forwarder WAD for vWii conversion

### ISSUE THREAD

**Issue by HTV04** [dborth/fceugx#393]
This is expanding on my previous issue. I couldn't reopen it for some reason. Do you have the base forwarder WAD from the installer so I can make it vWii compatible? I don't know how big the forwarder is, so I can't use its offsets to get the WAD from the installer.

**Comment by dborth** [comment]
I have both the forwarder code and the WADs they were injected into. Do code changes need to be made to the forwarder itself, to the WAD, or both?

**Comment by HTV04** [comment]
The WAD

**Comment by dborth** [comment]
[wads.zip](https://github.com/dborth/fceugx/files/3951302/wads.zip)

Okay, the WADs for the 3 emulators are attached.

**Comment by HTV04** [comment]
You used CustomizeMii to make the forwarders, right?

**Comment by dborth** [comment]
It's been a long time, but yes I believe I probably used CustomizeMii to inject the forwarder into the WAD. I didn't make the WADs myself someone else did and sent them to me. The source code for the forwarder itself is custom - not default from CustomizeMii. But I'm assuming you were able to keep the same forwarder intact?
[wiimc.zip](https://github.com/dborth/fceugx/files/3981876/wiimc.zip)

**Comment by HTV04** [comment]
If it's the 00000002.app file, then yes. I replaced the 00000001.app file with the NAND loader in each WAD.

I'll fix WiiMC and then test the WADs.

**Comment by dborth** [comment]
Any update on this?

**Comment by HTV04** [comment]
Oh my god, I'm so sorry! I forgot to test the WADs out. I'll test out the WADs soon. Hopefully they work!

**Comment by dborth** [comment]
Thanks!

**Comment by dborth** [comment]
thought I'd check in with you again on this 

any update @HTV04 ?

**Comment by HTV04** [comment]
Wow, I can't believe I keep forgetting to do this. I have the files ready, but I haven't tested them at all yet. I'm really sorry.

I guess I have no excuse now, I'm out of school for the next few weeks due to the coronavirus. I'll try to have it ready tomorrow. Please forgive me if I forget again.

**Comment by dborth** [comment]
Right here, it'll let you drag the files right in.

**Comment by dborth** [comment]
Thought I'd check in again and see if you've tried the files yet :)

**Comment by dborth** [comment]
I've put together vWii channels on my own, and built new forwarder channels to go along with them. Closing.

**Comment by HTV04** [comment]
Hey, I just noticed you've replied to this again. I'm really sorry for not updating you on this. I actually had the files sitting on my drive for a while, but I didn't want to release them until I fully tested them. I'm really sorry about this, and I'm glad you found a way to make working ones yourself.

So you know I'm not lying, I've attached all of the fixed WADs, dated March 19 (I had ones from a different computer that were older, but I decide to recompile them on my new computer).
[fixed.zip](https://github.com/dborth/fceugx/files/4707589/fixed.zip)

Maybe I should be more prepared when making a claim that I could make vWii WADs for you. I'm really sorry. The main fear I had that prevented me from releasing them was whether they worked at all. I didn't know you had a vWii, so I honestly feared that you wouldn't be able to test them, but you would release them and a lot of users would be angry at you for a stupid mistake I made. The reason I didn't test them myself was... Pure laziness. I had my Wii U completely unplugged and I didn't want to hook my Wii U up again. I could've tested them on my Wii (which was hooked up in my brother's room) but I knew they would not have worked as FIX94 explain in his description for the TinyNANDLoader.

I'm really sorry about this. Really. Hopefully next time (if there is), I won't take 10 years to help.
